NEWS.md

```diff
@@ -1,3 +1,7 @@
+# hint 1.29.1
```

Comment:
Re-used version number

Reply:
Oh sorry, it's not re-used is it, you've set patch version. OK, fair enough, I guess this really a bug fix.